react/searchResult.css

```diff
@@ -233,6 +233,12 @@
   overflow: hidden;
 }
 
+.filterQuantityBadgeDefault {
+  top: -0.7rem;
+  right: -0.8rem;
+  user-select: none;
```

Comment:
Can you explain the motivation of `user-select: none`?! I'm asking about it because it creates a bad UX and it let difficult for people with disabilities to copy this text.

Reply:
Hi @igorbrasileiro 

I pretty much copied the code and styles from MinicartIconButton, since I wanted to keep the feature as close as possible from components VTEX already have.

You can check the original source on the files below:

https://github.com/vtex-apps/minicart/blob/4ec5e58ca4bd4a30772ed11990023743aadfe15f/react/components/MinicartIconButton.tsx

https://github.com/vtex-apps/minicart/blob/4ec5e58ca4bd4a30772ed11990023743aadfe15f/react/styles.css

But since these icons will appear on mobile only, we don't really need this attribute.

Reply:
Ok, np!